Disable virtio transport in SUSE-provided config file

In SUSE distros, the default vhostmd config file has only enabled
one transport - vbd. Comment the recently added virtio transport.
Users can uncomment it and restart vhostmd if virtio transport is
desired, similar to the xenstore transport.

Index: vhostmd-1.1/vhostmd.xml
===================================================================
--- vhostmd-1.1.orig/vhostmd.xml
+++ vhostmd-1.1/vhostmd.xml
@@ -40,7 +40,7 @@ the logical && operator must be replaced
       <update_period>5</update_period>
       <path>/usr/sbin:/sbin:/usr/bin:/bin:/usr/share/vhostmd/scripts</path>
       <transport>vbd</transport>
-      <transport>virtio</transport>
+      <!-- <transport>virtio</transport> -->
       <!-- <transport>xenstore</transport> -->
     </globals>
     <metrics>
